package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@etherspot/prime-sdk",
-  "version": "1.6.4",
+  "version": "1.6.5",
```

Comment:
As its a breaking change, the version number should be 1.7.0

Reply:
I have made changes.